recipes/numba4jax/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "numba4jax" %}
+{% set version = "0.0.3" %}
+
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/numba4jax-{{ version }}.tar.gz
+  sha256: a88869874965bcaec2ce4deebe06a92dfa9102cc860702a2bad7aac8bec01ba6
+
+build:
+  number: 0
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: 
+    - {{ PYTHON }} -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools >=45
+    - setuptools_scm >=3.4
+    - toml
+  run:
+    - cffi >=1.12
+    - jax >0.2.16,<0.2.25
+    - jaxlib >0.1.70
+    - numba >=0.52,<0.55
+    - numpy ~=1.18
```

Comment:
This numpy pin is pretty desrtuctive. Why won't newer numpy versions work?

Reply:
This means numpy>=1.18, <2

Reply:
Indeed, i think I am not mistaken https://packaging.python.org/guides/distributing-packages-using-setuptools/#semantic-versioning-preferred

This follows semantic versioning.
Though it's probably useless because most python world does not follow semantic versioning...

I don't mind removing those, however I find them useful: the package is tested with those bounds, so this guarantees it works

Reply:
Ahhhhhhh Sorry! My brain did not compute that properly. Thank you!